main/zoe/guide/proposal.md

```diff
@@ -1,41 +1,68 @@
 
-## Structure of the proposal
+## Invitations
+
+`Invitations` are a special case of ERTP `payments`. They are linked to a specific contract `instance`, and
+having one gives you the right to participate in that contract `instance`, for example, by making offers in it.
+
+There are two main ways for contract users to get an `invitation`:
+- If you create the contract `instance`, you get a special creator `invitation`.
+- Someone who holds the right objects has created an `invitation` for a contract `instance` and given it to
+  you in some way. This could've been by sending it to you, posting it on a public online location, etc. It
+  doesn't matter (nor does Zoe specify or have any requirements) how or why it got to you, only that you have it.
+
+## Proposals
 
 Proposals are objects with `give`, `want` and `exit` keys.
 
-`give` and `want` utilize keywords defined by the contract.
+```js
+const myProposal = harden({
+  give: { Asset: quatloosAmountMath.make(4)},
+  want: { Price: moolaAmountMath.make(15) },
+  exit: { 'onDemand'
+})
+```
+`give` and `want` use keywords defined by the contract.
 Keywords are unique identifiers per contract, that tie together the proposal,
 payments to be escrowed, and payouts to the user.
-In the example below, "Asset" and "Price" are the keywords. In an auction, however,
-the keywords might be "Asset" and "Bid".
-
-`exit` determines when the user can cancel their offer. The possible kinds are:
-1. 'onDemand': the user can cancel on demand
-2. 'afterDeadline': the user's offer is automatically cancelled after
-   a deadline. This option requires additional properties: a timer
-   object and a deadline.
-3. 'waived': the user agrees that they can't cancel and they are
-   relying entirely on the smart contract to complete their offer
-   promptly.
+In the example above, `Asset` and `Price` are the keywords. However, in an auction contract,
+the keywords might be `Asset` and `Bid`.
+
+The `quatloosAmountMath.make(4)` is just making an ERTP `amount`, or description of digital assets.
+In this case, 4 of our imaginary Quatloos currency. `moolaAmountMath.make(15)` is making an `amount` of 15 of our imaginary Moola currency. 
 
+**Note**: It's important to understand that `amounts` are just descriptions of assets with no
+intrinsic value. `payments` hold actual digital assets.
+
+`exit` determines how an offer can be can cancelled:
+- `'onDemand'`: (Default) Whenever the user wants.
+- `'waived'`: The user cannot cancel, relying on the contract to finish the offer.
+- `'afterDeadline'`: Cancelled automatically after a deadline. This requires two
+  more properties, a `timer` object and a deadline value.
+
+## Escrowed Payments
+
+Using the same keywords as your `proposal`, you must specify a `PaymentKeywordRecord`.
+This is a record with the keywords as keys, and `payments` containing digital assets as
+values. Zoe escrows these `payments` until the offer is completed or rejected. 
```

Comment:
```suggestion
values. Zoe escrows these `payments` on behalf of this seat until the offer is completed or rejected or the assets are reassigned to another seat. 
```

Reply:
Changed, but I didn't have time to upgrade this to cover seats (the original only did proposals; offers was in-between). So could "seats" be changed to "offers" here for the moment until this can be upgraded to cover seats?

Reply:
sure.